config/configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties

```diff
@@ -853,6 +853,19 @@
 # DEFAULT: Empty, no value
 # portal.include.extrahead=<script type="text/javascript" language="JavaScript" src="/kaltura/javascript/kaltura-display.js"></script>
 
+# Allows for setting the cdn path for static portal content
+# DEFAULT: Empty
+# portal.cdn.path=http://nightly2.sakaiproject.org
+
+# Allows for setting a version for static portal content
+# DEFAULT: uses a git SHA from the build
+# portal.cdn.version=c3cc2a0
```

Comment:
it would be cooler if the build actually wrote the git sha into default.sakai.properties. lol.